controllers/gitopsservice_controller.go

```diff
@@ -344,6 +349,15 @@ func (r *ReconcileGitopsService) reconcileDefaultArgoCDInstance(instance *pipeli
 			defaultArgoCDInstance.Spec.NodePlacement.NodeSelector = argocdutil.AppendStringMap(defaultArgoCDInstance.Spec.NodePlacement.NodeSelector, instance.Spec.NodeSelector)
 		}
 	}
+	if len(instance.Spec.NodeSelector) > 0 {
```

Comment:
Again, should this NodeSelector related code be part of this PR?

Reply:
Thanks for the catch. This should be fixed now.